src/include/processor/operator/physical_operator.h

```diff
@@ -116,6 +116,7 @@ class PhysicalOperator {
 
     inline virtual bool isSource() const { return false; }
     inline virtual bool isSink() const { return false; }
+    inline virtual bool parallel() const { return true; }
```

Comment:
Consider renaming `parallel` to `isParallel`. I think the latter one reads more naturally.